CONTRIBUTING.md

```diff
@@ -265,10 +279,3 @@ For more information about possible improvements, see
 ### BufferManager
 
 BufferManager is responsible for the syncing of buffers and windows between vscode and nvim.
-
-## Maintenance
```

Comment:
do not remove this section. we need a place to keep notes about doing releases, etc. "style and tools" is not the right place for that.